packages/tools/replay-tool/src/helpers.ts

```diff
@@ -48,6 +48,26 @@ export function getNormalizedFileSnapshot(snapshot: IFileSnapshot): IFileSnapsho
     return normalizedSnapshot;
 }
 
+/**
+ * The packageVersion of the snapshot could be different from the reference snapshot. Replace all package
+ * package versions with X before we compare them. This is how it will looks like:
+ * Before replace - "{\"type\":\"https://graph.microsoft.com/types/map\",\"packageVersion\":\"0.28.0-214\"}"
+ * After replace  - "{\"type\":\"https://graph.microsoft.com/types/map\",\"packageVersion\":\"X\"}"
+ */
+const packageVersionRegex = /\\"packageversion\\":\\".+\\"/gi;
+const packageVersionPlaceholder = "\\\"packageVersion\\\":\\\"X\\\"";
+
+const normalize = (snapshot: IFileSnapshot) =>
+    JSON.parse(
+        JSON.stringify(snapshot, undefined, 2)
+            .replace(packageVersionRegex, packageVersionPlaceholder)
+            // We changed routing in 0.36, old ops & snapshots use old schema.
+            // Snapshots regeneration does not solve it, as handles are not adjusted to new schema on read/write cycle.
+            // Replace new usage to old schema to make it possible to compare snapshots
+            .replace(/\/_channels/g, "")
```

Comment:
I think we need to convert the absolute path of handles from old format to new otherwise GC will be broken.
For example, a snapshot in old format has a handle to datastore1 serialized as `/datatstore1`. When we run GC and generate GC data, the GC node for this data store will have its id as `/_channels/datastore1`. So, GC will incorrectly think that datastore1 is unreferenced because it won't be able to find a node with id `/datastore1`.